pkg/providers/aws/provider_postgres.go

```diff
@@ -214,6 +219,88 @@ func (p *AWSPostgresProvider) createRDSInstance(ctx context.Context, cr *v1alpha
 	}}, croType.StatusMessage(fmt.Sprintf("creation successful, aws rds status is %s", *foundInstance.DBInstanceStatus)), nil
 }
 
+// Tags RDS resources
+func (p *AWSPostgresProvider) TagRDSPostgres(ctx context.Context, cr *v1alpha1.Postgres, foundInstance *rds.DBInstance) (croType.StatusMessage, error) {
+	logrus.Infof("Adding Tags to RDS instance %s", *foundInstance.DBInstanceIdentifier)
+	// get the environment from the CR
+	defaultOrganizationTag, exists := os.LookupEnv("DEFAULT_ORGANIZATION_TAG")
+	if exists {
+		// Print the value of the environment variable
+		p.Logger.Infof("env var DEFAULT_ORGANIZATION_TAG : %s", defaultOrganizationTag)
```

Comment:
i think we can remove this log, unless there is a plan to perform some other task in this block?